src/IROperator.cpp

```diff
@@ -567,7 +567,7 @@ Expr halide_erf(Expr x_full) {
     return result;
 }
 
-Expr raise_to_integer_power(Expr e, int p) {
+Expr raise_to_integer_power(Expr e, int64_t p) {
```

Comment:
Immediates can now be 64-bit. See: https://github.com/halide/Halide/blob/master/src/IROperator.h#L1191